### ISSUE TITLE
If you log too much you get "too many requests" errors

### ISSUE THREAD

**Issue by s-baumann** [Arkoniak/Telegram.jl#18]
```
using ConfigEnv
dotenv()


using Telegram, Logging, LoggingExtras

# Setting up Telegram for logging.
tg = TelegramClient()
tg_logger = TelegramLogger(tg; async = false)
demux_logger = TeeLogger(
    MinLevelLogger(tg_logger, Logging.Info),
    ConsoleLogger()
)
global_logger(demux_logger)

counter = 0
while counter < 10000
    @info "counter is $(counter)"
    global counter += 1
end
```
After 100 this gives me 

```
ERROR: LoadError: HTTP.ExceptionRequest.StatusError(429, "POST", "/BOTINFO/sendMessage", HTTP.Messages.Response:
"""
HTTP/1.1 429 Too Many Requests
Server: nginx/1.18.0
Date: Thu, 03 Nov 2022 23:26:19 GMT
Content-Type: application/json
Content-Length: 111
Connection: keep-alive
Retry-After: 10
Strict-Transport-Security: max-age=31536000; includeSubDomains; preload
Access-Control-Allow-Origin: *
Access-Control-Expose-Headers: Content-Length,Content-Type,Date,Server,Connection

{"ok":false,"error_code":429,"description":"Too Many Requests: retry after 10","parameters":{"retry_after":10}}""")
Stacktrace:
```

It is possible to set an option so that it continues without errors in this case? In some applications it doesnt make sense to crash a program due to not being able to log it.

**Comment by Arkoniak** [comment]
Yes, this is good remark, it should be wrapped in `try catch` block.

**Comment by Arkoniak** [comment]
Thank you for this issue, it is fixed in v1.1.4
